JudicialCouncilOfCalifornia/trialcourt
# [DCAS-164] [DCAS-200] - Better News View alterations and option to override default behavior of section filtering

[DCAS-164] - Better News View alterations to adjust for sectioned content.
[DCAS-200] - Option to override default behavior in how section filtering is applied to views with exposed forms.

**Instructions**:
- Visit section settings page and set the News list view/display to be overridden contextually (2nd set of options in views settings)

**Testing**:
- Confirm that News Announcments pages for each section no longer has the Section exposed filter, yet still displays proper items.
- Confirm that the News listing on each Section home page displays proper amount of sectioned items
